app/serializers/main_serializer.rb

```diff
@@ -0,0 +1,17 @@
+class MainSerializer < ActiveModel::Serializer
+  attributes :id, :content, :votes_count, :voted, :created_at, :updated_at, :user
+
+  def voted
+    object.votes.voted?(object.class.to_s, object.id, object.user)
+  end
+
+  def user
```

Comment:
Just a though; we can replace this with `belongs_to :user` to leverage the `UserSerializer`